package.json

```diff
@@ -160,7 +160,7 @@
     "bowser": "^1.7.1",
     "bundle-loader": "^0.5.4",
     "cbioportal-clinical-timeline": "^0.3.82",
-    "cbioportal-frontend-commons": "^0.5.66",
+    "cbioportal-frontend-commons": "^0.5.67-beta.0",
```

Comment:
we don't need to manually update the library to a beta version. it is okay to release 0.5.66 with the updated library dependencies. let's revert this change.